### DISCUSSION TITLE
Are blog posts still available?

### DISCUSSION THREAD

**Discussion by h-hmz** [benbjohnson/wtf#82] [General]
Hello!

I was really eager to dive into the code and its companion blog posts, however I couldn't find them on the website apart from the initial one: https://www.gobeyond.dev/tag/wtf-dial/

Have they been deleted or relocated?

Thanks!

**Comment by itaranto** [comment]
Here's a couple:

https://www.gobeyond.dev/structuring-applications/
https://www.gobeyond.dev/standard-package-layout/
